freqtrade/constants.py

```diff
@@ -600,6 +600,7 @@
                         "policy_type": {"type": "string", "default": "MlpPolicy"},
                         "net_arch": {"type": "array", "default": [128, 128]},
                         "randomize_starting_position": {"type": "boolean", "default": False},
+                        "progress_bar": {"type": "boolean", "default": False},
```

Comment:
should we default this to true? 
it seems to be a nice feature. if someone is bothered by it, stopping it would still be an option ? 
just a thought ... i'm fine with it either way

Reply:
I would vote for defaulting to true, I will make the change